dm/master/coordinator/coordinator.go

```diff
@@ -189,8 +189,8 @@ func (c *Coordinator) HandleStartedWorker(w *Worker, cfg *config.MysqlConfig, su
 		c.taskConfigs[cfg.SourceID] = *cfg
 	} else {
 		w.SetStatus(WorkerFree)
-		delete(c.pendingtask, cfg.SourceID)
 	}
+	delete(c.pendingtask, cfg.SourceID)
```

Comment:
Is `pendingWorker` or another name with `Worker` more meaningful? the same with `taskConfigs`?

Reply:
How about 'pendingRequests' ?